src/components/Chart.jsx

```diff
@@ -55,19 +55,15 @@ class Chart extends Component {
                         </RenderInsideChart>
                         <RenderInsideChart>
                             <div className="cq-top-ui-widgets">
-                                <ChartTitle />
                                 <AssetInformation />
                                 <ComparisonList />
                             </div>
                         </RenderInsideChart>
+                        <ChartTitle />
                         <ChartControls />
                         <Crosshair />
                         <div className="chartContainer primary"> </div>
                         <Loader />
-                        {!isChartAvailable &&
```

Comment:
you didn't merge `dev` properly. I'd recommend using something like [Meld](http://meldmerge.org/) or similar to handle conflicts